src/js/services/werewolf.js

```diff
@@ -42,4 +42,6 @@ export default new class werewolfService {
     getScript(deck, lang = 'en') { //TODO: better defaults
         return brain.getScriptFromDeck(deck, lang);
     }
+
+    mode = brain.getModes;
```

Comment:
I think it would better just import `werewolf-brain` where we need to use `brain.getModes`. Don't know why but this property here too forever alone or something. What do you think guys? @wOvalle @mamodom